src/java/org/apache/cassandra/auth/Roles.java

```diff
@@ -38,7 +39,8 @@
 
     private static final Role NO_ROLE = new Role("", false, false, Collections.emptyMap(), Collections.emptySet());
 
-    private static RolesCache cache;
+    @VisibleForTesting
+    static RolesCache cache;
```

Comment:
It looks like this `cache` does not need to be mutable. You may make it final - it is initalized in the static block anyway

Reply:
I think it's not final b/c there are a couple tests that re-init the cache by calling `initRolesCache`